packages/babel-helper-create-class-features-plugin/src/features.js

```diff
@@ -13,7 +13,7 @@ const featuresSameLoose = new Map([
   [FEATURES.privateMethods, "@babel/plugin-proposal-private-methods"],
   [
     FEATURES.privateIn,
-    "@babel/plugin-proposal-private-private-property-in-object",
+    "@babel/plugin-proposal-private-property-in-object",
   ],
```

Comment:
```suggestion
```

Reply:
Guess we should have a lint fix workflow.

Reply:
Yeah that would help

Reply:
An option could be to hook ESLint up to Prettier in the lint-staged hook for auto clean up with something to the effect of ( not exactly, but for reference ):

```
"*.{js,jsx,ts,tsx,css,scss,md}": [
            "prettier --ignore-path ./.eslintignore --write"
        ]
```

I guess this might increase commit time slightly so probably want to take with a grain of salt & weigh tradeoffs

Reply:
We have a CI job that automatically updates some fixtures when they need to be updated (https://github.com/babel/babel/actions/runs/615369743/workflow), we can do something similar.